src/process_ext.rs

```diff
@@ -0,0 +1,63 @@
+use std::fs::write;
+use std::os::unix::process::CommandExt;
+use std::path::PathBuf;
+use std::process;
+use std::process::Command;
+
+use crate::Cgroup;
+
+/// Extensions to default `std::process::Command` capabilities.
+pub trait CgroupsCommandExt {
+    /// Sets cgroups for the process to be put into before execution of that process starts.
+    ///
+    /// # Examples
+    /// ```
+    /// use std::process::Command;
+    /// use cgroups_rs::{Cgroup, CgroupPid};
+    /// use cgroups_rs::cgroup_builder::CgroupBuilder;
+    /// use cgroups_rs::MaxValue::Value;
+    /// use cgroups_rs::process_ext::CgroupsCommandExt;
+    /// use std::cell::Ref;
+    ///
+    /// let mut process = Command::new("ls");
+    ///
+    /// let hierarchy = cgroups_rs::hierarchies::auto();
+    /// let cgroup_name = "test";
+    /// let cg: Cgroup = CgroupBuilder::new(cgroup_name)
+    ///         .cpu()
+    ///         .cpus("1".to_owned())
+    ///         .shares(100)
+    ///         .done()
+    ///         .build(hierarchy);
+    /// let mut child = process.cgroups(&[&cg])
```

Comment:
Can you add a call to `.arg()` (maybe `.arg("-al")`?) in this example. It's useful to show how that works, particularly since I think you'd need to add the call to `.arg()` here rather than above where you define `process`.